commands/utils/params.go

```diff
@@ -19,7 +18,8 @@ const (
 	FrogbotConfigFile = "frogbot-config.yml"
 )
 
-var configRelativePath = filepath.Join(".", ".jfrog", FrogbotConfigFile)
+// Possible Config file path's to Frogbot Management repository
+var frogbotConfigPath = filepath.Join(".", ".frogbot", FrogbotConfigFile)
```

Comment:
I think the "." is redundancy
```suggestion
var frogbotConfigPath = filepath.Join(".frogbot", FrogbotConfigFile)
```

Reply:
done